src/app/components/account/account-list-item-layout.tsx

```diff
@@ -54,35 +56,35 @@ export function AccountListItemLayout(props: AccountListItemLayoutProps) {
       onClick={onSelectAccount}
       {...rest}
     >
-      <Flag align="middle" img={avatar} spacing="base" width="100%">
-        <Stack spacing="extra-tight">
+      <Flag align="middle" img={avatar} spacing="space.04" width="100%" mr="space.04">
+        <Stack gap="space.01">
           <HStack alignItems="center" justifyContent="space-between">
-            <Stack alignItems="center" isInline space="tight">
+            <HStack alignItems="center" gap="space.02">
               {accountName}
               {isActive && <CheckmarkIcon />}
-            </Stack>
+            </HStack>
             {isLoading ? (
               <Spinner
                 position="absolute"
                 right={0}
                 top="calc(50% - 8px)"
-                color={color('text-caption')}
+                color={token('colors.accent.text-subdued')}
                 size="18px"
               />
             ) : (
               balanceLabel
             )}
           </HStack>
-          <Stack alignItems="center" spacing="6px" isInline whiteSpace="nowrap">
+          <HStack alignItems="center" gap="space.02" whiteSpace="nowrap">
             <CaptionDotSeparator>
               <styled.span textStyle="caption.02">
-                {truncateMiddle(stxAddress, isNarrowViewport ? 3 : 4)}
+                {truncateMiddle(stxAddress, isBreakpointSm ? 4 : 3)}
```

Comment:
I don't think we need this `isBreakpointSm` part here. It's changing how much we truncate the address based on the viewport. 

It doesn't really make much difference as we can handle a longer address in each section. 

Current code:
![Screenshot 2023-10-04 at 14 00 20](https://github.com/leather-wallet/extension/assets/2938440/daf73d0d-8a3a-4ab4-adc4-1e285c7e37ba)

Setting it to `4` and no `isBreakpointSm`
![Screenshot 2023-10-04 at 14 01 14](https://github.com/leather-wallet/extension/assets/2938440/c68800e1-c876-4012-adc9-5b63fac70bba)

Reply:
@kyranjamie let me know what you think here

Reply:
OK, I'll remove it soon 👍